craft_parts/lifecycle_manager.py

```diff
@@ -80,7 +80,7 @@ class LifecycleManager:
         enabled. The first partition must be "default" and all partitions must be
         lowercase alphabetical.
     :param custom_args: Any additional arguments that will be passed directly
-        to :ref:`callbacks<callbacks>`.
+        to callbacks.
```

Comment:
for the life of me I could not get a reference to the auto-generated docs to work